### ISSUE TITLE
testj1939: socket(j1939): Invalid argument.

### ISSUE THREAD

**Issue by miteshdube** [linux-can/can-utils#312]
I am following this [documentation](https://github.com/linux-can/can-utils/blob/master/can-j1939-kickstart.md). But for `modprobe` I'm getting below error.
```
~ # modprobe can-j1939
modprobe: FATAL: Module can-j1939 not found in directory /lib/modules/4.4.207
```
`testj1939` command shows below error. 
```
~ # testj1939 -B -r can0
testj1939: socket(j1939): Invalid argument
```

I think this error is due to not loading of kernel module for `can-j1939`, if I am correct then can anyone please provide me source code for `can-j1939`.

**Comment by marckleinebudde** [comment]
j1939 is included in the Linux kernel since v5.4. Consider updating to a newer kernel.

**Comment by miteshdube** [comment]
@marckleinebudde Can we back port the CAN-j1939 driver from Linux5.4 to Linux 4.4.207?

**Comment by marckleinebudde** [comment]
Hey @miteshdube,

sure, it's Open Source. However, using the latest mainline kernel is preferred.

**Comment by prajwal-mulkalwar** [comment]
Facing same issue in Jetson Nano
4.9.201-tegra

**Comment by marckleinebudde** [comment]
j1939 is included in the Linux kernel since v5.4. Consider updating to a newer kernel.
